### ISSUE TITLE
collector-http.sh possible bug

### ISSUE THREAD

**Issue by AntonioAmore** [Norconex/collector-http#59]
The current log4j configuration path at the collector-http.sh (line 6) is pointed as:
```sh
log4jconfiguration="file:///${ROOT_DIR}/classes/...
```

For me it works without log4j exceptions when I edited it such way:
```sh
log4jconfiguration="file:${ROOT_DIR}/classes/...
```

OS: Debian

**Comment by essiembre** [comment]
I reproduced on Fedora as well.  The fix will be in next release.  Thanks for reporting it.

**Comment by essiembre** [comment]
The fix is in the 2.1.0-SNAPSHOT release.

**Comment by essiembre** [comment]
[Norconex HTTP Collector 2.1.0](http://www.norconex.com/collectors/collector-http/download) was released. Closing.
